src/containers/auth/resetPassword/index.jsx

```diff
@@ -0,0 +1,137 @@
+                    const { status, data } = await resetPassword({ linkId, password, passwordConfirm });
+                    if (data.status === 200) {
+                        this.setState({
+                            status,
+                            success: (status < 300)
+                        });
+                    } else {
+                        this.setState({
+                            message: data.message
+                        });
+                    }
+                }
+            } else {
+                this.setState({
+                    message: 'Passwords not equal'
+                });
+            }
+        } else {
+            this.setState({
+                message: 'Enter password'
+            });
+        }
+
+
+    }
+
+    inputHandler = (e) => {
+        this.setState({
+            [e.target.name]: e.target.value,
+        });
+    }
+
+    render() {
+        const { status, success, message } = this.state;
+        const {
+            input,
+            btn,
+            form,
+            row,
+            rowFlexEnd,
+            wrapperF,
+            errorMessage,
+            successMessage,
+            big_size_message,
+        } = styles;
+
+        return (
+            <div className={wrapperF}>
+                {(status) && (status < 300) && (!success) &&
+                    (<form
+                        name="resetPasswordForm"
+                        className={form}
+                    >
+                        <div className={row}>
+                            <input
+                                type="password"
+                                name="password"
+                                id="password"
+                                placeholder='Enter Password'
+                                onChange={this.inputHandler}
+                                className={input}
+                                required
+                            />
+                        </div>
+                        <div className={row}>
+                            <input
+                                type="password"
+                                name="passwordConfirm"
+                                id="passwordConfirm"
+                                placeholder='Confirm Password'
+                                onChange={this.inputHandler}
+                                className={input}
+                                required
+                            />
+                        </div>
+                        <p className={`${errorMessage} `}>&nbsp;{!!message && message}</p>
+                        <div className={`${row} ${rowFlexEnd}`}>
+                            <button type='submit' className={btn} onClick={this.sendReset} >Push</button>
+                        </div>
+
+                    </form>)}
+                {(status) && (!success) && (status > 300) && (<div className={`${big_size_message} ${errorMessage}`}>Something is wrong</div>)}
```

Comment:
Make this logic easier pls. It is hard to understand

Reply:
done